bundles/org.smarthomej.binding.notificationsforfiretv/src/main/java/org/smarthomej/binding/notificationsforfiretv/internal/NotificationsForFireTVHandler.java

```diff
@@ -0,0 +1,125 @@
+
+    private NotificationsForFireTVConfiguration config = new NotificationsForFireTVConfiguration();
+
+    public NotificationsForFireTVHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        config = getConfigAs(NotificationsForFireTVConfiguration.class);
+
+        updateStatus(ThingStatus.ONLINE);
+    }
+
+    public boolean sendNotification(@Nullable String msg, @Nullable String filename, @Nullable String filename2) {
+        try {
+            // CREATE CONNECTION
+            NotificationsForFireTVConnection notificationsForFireTVConnection = new NotificationsForFireTVConnection(
+                    config.ip, config.port);
+            // ADD FORM FIELDS
+            notificationsForFireTVConnection.addFormField(TYPE, String.valueOf(config.type));
+            notificationsForFireTVConnection.addFormField(TITLE, config.title);
+            notificationsForFireTVConnection.addFormField(MSG, msg != null ? msg : "");
+            notificationsForFireTVConnection.addFormField(DURATION, String.valueOf(config.duration));
+            notificationsForFireTVConnection.addFormField(FONTSIZE, String.valueOf(config.fontsize));
+            notificationsForFireTVConnection.addFormField(POSITION, String.valueOf(config.position));
+            notificationsForFireTVConnection.addFormField(BKCOLOR, config.bkgcolor);
+            notificationsForFireTVConnection.addFormField(TRANSPARENCY, String.valueOf(config.transparency));
+            notificationsForFireTVConnection.addFormField(OFFSET, String.valueOf(config.offset));
+            notificationsForFireTVConnection.addFormField(OFFSETY, String.valueOf(config.offsety));
+            notificationsForFireTVConnection.addFormField(APP, config.app);
+            notificationsForFireTVConnection.addFormField(FORCE, String.valueOf(config.force));
+            notificationsForFireTVConnection.addFormField(INTERRUPT, String.valueOf(config.interrupt));
+            // ADD FILES
+            if (filename != null) {
+                File file = new File(filename);
+                if (!file.exists()) {
+                    throw new IOException("File doesn't exist: " + filename);
+                }
+                notificationsForFireTVConnection.addFilePart(FILENAME, file);
+            }
+            if (filename2 != null) {
+                File file2 = new File(filename2);
+                if (!file2.exists()) {
+                    throw new IOException("File doesn't exist: " + filename2);
+                }
+                notificationsForFireTVConnection.addFilePart(FILENAME2, file2);
+            }
+            // POST FORM
+            notificationsForFireTVConnection.finish();
+
+            return true;
+        } catch (Exception e) {
+            logger.error("Unable to connect: {}", e.getMessage());
```

Comment:
```suggestion
            logger.warn("Unable to connect: {}", e.getMessage());
```

ERROR is reserved for framework issues that affect the stability of the whole system

Reply:
done, first missed it but changed it now